src/app.js

```diff
@@ -1,21 +1,30 @@
 import { div } from '@cycle/dom'
+import isolate from '@cycle/isolate'
 import xs from 'xstream'
+import { Instrument } from './components'
 
-export function App() {
-  const vtree$ = xs.of(
-    div('My Awesome Cycle.js app'),
-  )
+export function App({ DOM$ }) {
+  const randomFrequency = () => Math.round(Math.random() * 1000) + 200
 
-  const music$ = xs.periodic(200).map(() => {
-    return {
-      frequency: (Math.random() * 1000) + 300,
-      time: (Math.random() * 1000) + 200,
-    }
-  })
+  const instruments = [
+    { frequency: randomFrequency() },
+    { name: 'guitare', frequency: randomFrequency() },
+    { name: 'piano', frequency: randomFrequency() },
+    { name: 'ocarina', frequency: randomFrequency() },
+    { name: 'tamtam', frequency: randomFrequency() },
+  ].map(instrument => isolate(Instrument)({ DOM$, props$: xs.of(instrument) }))
```

Comment:
I don't think `intrument` is a good variable name since this is rawData and not an instance of `Intrument`